api/candlepin-api-spec.yaml

```diff
@@ -236,6 +236,27 @@ paths:
               schema:
                 $ref: '#/components/schemas/ExceptionMessage'
 
+  /status:
+    get:
+      description: Returns status of the server
+      tags:
+        - Status
+      operationId: status
+      security: []
+      responses:
+        200:
+          description: ''
```

Comment:
With the latest rebase, the description got accidentally reverted back to an empty string it seems. Once it's added back it should be good to merge